ruby-builder.js

```diff
@@ -15,17 +16,40 @@ export function getAvailableVersions(platform, engine) {
 
 export async function install(platform, ruby) {
   const rubyPrefix = await downloadAndExtract(platform, ruby)
+  return rubyPrefix
+}
+
+export async function setupPath(rubyPrefix, ruby) {
+  const rubyPath = path.join(rubyPrefix, 'bin', 'ruby')
+  const gemUserDir = cp.execSync(`${rubyPath} -e "puts Gem.user_dir"`).toString().trim()
```

Comment:
Why do you add the Gem.user_dir here?
AFAIK nobody bothers with `gem install --user-dir` in CI, they expect they can just `gem/bundle install`.

Reply:
> Why do you add the Gem.user_dir here?

The `"#{Gem.user_dir}/bin"` directory.  Just because someone, someday, will ask about it, and it's not hard to include...

Reply:
I prefer to keep the logic simpler here and not try to guess what someone, someday, will ask.

Reply:
Also shelling out to `ruby` should be avoided, that will slow down every usage of the action. On JRuby it can take easily 1 second just to shell out to print `Gem.user_dir`.

Reply:
See https://github.com/rubygems/rubygems/pull/2847

Not implemented yet, but...

Reply:
Thanks for the link. I don't like to speculate on the future though, and I want to keep this action as simple as possible, simply because otherwise I won't have time to maintain it.